conf/pacific/upgrades/upgrade_from4x_small_cluster.yml

```diff
@@ -37,7 +37,6 @@ globals:
          - iscsi-gw
          - client
 
-           #     node6:
-           #       role:
-           #         - nfs
-           #     nfs upgrade is blocked till 5.1
+     node6:
```

Comment:
Likewise, let's create another one for 5.1 test runs. As the pipeline scripts are different for each version, the new change wouldn't break the execution for both the releases.